modules/vaos/spec/requests/v2/referrals_spec.rb

```diff
@@ -48,6 +48,47 @@
         expect(first_referral['attributes']).to have_key('type_of_care')
       end
     end
+
+    context 'when a configuration error occurs' do
+      # Note we only test this once as the code is the same for both endpoints
+      let(:jwt_error) { Common::JwtWrapper::ConfigurationError.new('Configuration error occurred') }
+      let(:config_error) { VAOS::Exceptions::ConfigurationError.new(jwt_error, 'CCRA') }
+
+      before do
+        sign_in_as(user)
+        allow(service_double).to receive(:get_vaos_referral_list).and_raise(config_error)
+      end
+
+      it 'returns 503 Service Unavailable with properly formatted error response' do
+        get '/vaos/v2/referrals'
+
+        expect(response).to have_http_status(:service_unavailable)
```

Comment:
Note: Here is the key change. The API error no longer has exception information. It is just a generic message, but the real error was logged.